ibis/expr/api.py

```diff
@@ -4418,11 +4418,32 @@ def _table_view(self):
     return new_view.to_expr()
 
 
-def _table_drop(self, fields):
+def _table_drop(self: ir.TableExpr,
+                fields: Union[str, List[str]]
+                ) -> ir.TableExpr:
+    """
+    Remove one or more fields from a table.
+
+    _table_drop(table, "a-string") is equivalent
```

Comment:
you can remove these 2 lines, they are extraneous

Reply:
I don't understand. You asked me: "hmm there isn't any doc-string, can you add one, also type fields."

Where should the doc-string be added?

Reply:
Docstring is fine where it is. Jeff's comment is that you can remove those two lines about the two calls being equivalent.